File: C++ Programming/problems on recursion/problem5.cpp
/*
5.Write a recursive program which display below pattern.  
	
	Input : 6
	
	Output :   a b c d e f 
	
	Author :Milind Nalawade
	
	Date : 25/08/2021
	
	Language : C++ programming
	
*/
using namespace std;
#include<iostream>

char ch='a';

//Logic
	
void Print(int no)
{
	if(no==0)
	{
		return;
	}
	else
	{
		cout<<" "<<ch;
	}
	ch++;
	no--;
	Print(no);
}	

//Entry point function
int main()
{
	int no=0;
	cout<<"\n Input : ";
	cin>>no;
	cout<<"\n Output : ";
	Print(no);
	cout<<"\n";
	return 0;
}	

File: Python Programming/problems on recursion/problem5.py
'''
5.Write a recursive program which display below pattern.  
	
	Input : 6
	
	Output :   a b c d e f 
	
	Author :Milind Nalawade
	
	Date : 25/08/2021
	
	Language : Python 3.9.0

'''
#====
#Logic
#====
ch='a'
def Print(no):
	global ch
	if no==0:
		return
	else:
		print(" ",ch,end="")
	ch=chr(ord(ch)+1)
	no-=1
	Print(no)
	
#=============
#Entry point function
#=============
def main():
	no=int(input("\n Input : "))
	print("\n Output : ",end="")
	Print(no)
	print()
	
#=====
#starter
#=====
if __name__=="__main__":
	main()